docs/sources/developers/plugins/add-support-for-variables.md

```diff
@@ -58,3 +58,97 @@ For data sources, you need to use the [getTemplateSrv]({{< relref "../../package
      return { data };
    }
    ```
+
+## Add support for variable queries to your data source
```

Comment:
Do you mean add support for _query variables_ to your data source? Or for using variables in queries? Unclear.

Also, will this be just template variables (defined by the user) or does this include global variables as well?

Reply:
I was going back and forth on this. The feature is called Query variables, but from the plugin's perspective it's about adding support for "Query variables queries".

This is only for creating template variables. I don't believe plugins can create global variables themselves.

Reply:
How about, "Add support for query variables to your data source"?